cost_calculator/templates/cost_summary.html

```diff
@@ -42,7 +52,8 @@
     </thead>
   </table>
   <p>
-    <strong>Note:</strong> Nodeless cost is projected based on cpu and memory resource requests and limits for the applications running on the cluster. Projected cost uses on-demand instance price by default. Spot projections coming soon!
+    <strong>Note:</strong> Nodeless cost is projected based on cpu and memory resource requests and limits for the applications running on the cluster. Projected cost uses on-demand instance price by default. If row is yellow, it means that resource requirements are not defined for given pod.
+      This may falsify calculations and it's considered bad practice. Please set resource requirements for those pods to get a more giaccurate forecast!
```

Comment:
I'd rephrase this line like this:

> This may result in inaccurate results in the calculator and not recommended for production systems.

There's also a typo "giaccurate" -> "accurate".

Reply:
thanks!